### DISCUSSION TITLE
Welcome to dateSelector Discussions!

### DISCUSSION THREAD

**Discussion by o221** [o221/dateSelector#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tssmith1996-pmg** [comment]
Hey o221, this is such a greate alternative date filter! One thing that I'm having trouble with though is that there is no option to turn off tooltips (so, for example, the date range tooltip doesn't cover the shortcut buttons if they are below the date inputs). Do you have any plans to add this sort of functionality?

**Comment by o221** [comment]
Hi tss

I have done long tooltips but not no tooltips. It should be quite easy to
do.

**Reply by o221** [reply]
Change done. You can now hide the tooltip, show standard tooltip or show detailed tooltip with shortcut keys.

**Comment by rskibicki** [comment]
In Power BI the dates seem to love the year 1969.  Once I hit the calendar the year is always in 1969 in the from and to windows.  Anyway to change that? The program is interesting if the calendar will work

**Reply by o221** [reply]
We have not experienced that situation. Does the sample pbix work for you? Sounds like your date field may be the issue. Can you provide a pbix with your date field so we can check?
